docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tuning and Best Practices for Rancher at Scale
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale"/>
+:docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md
+</head>
+
+
+This guide describes best practices and tuning approaches to scale Rancher setups, and associated challenges with doing so. As systems grow performance will naturally reduce, but there are steps that can be taken to minimize the load put on Rancher, as well as optimize Rancher's ability to manage larger infrastructures.
+
+## General Guidelines on Optimizing Rancher's Performance
+* Keep Rancher up to date with patch releases. Performance improvements and bug fixes are made continuously, and the latest release incorporates the largest set ofperformance related development, experience and feedback from many users.
```

Comment:
I like the new wording, thanks!

Just out of curiosity: is the use of "we" and "you" generally considered OK for technical documentation? I think I remember some projects ago there was a recommendation to avoid use of such pronouns in favor of passive forms. As a non-native speaker I of course do not have an opinion on my own - those nuances are way too subtle for my sensitivity to even detect 😉

Reply:
> @moio 
> I like the new wording, thanks!
> 
> Just out of curiosity: is the use of "we" and "you" generally considered OK for technical documentation? I think I remember some projects ago there was a recommendation to avoid use of such pronouns in favor of passive forms. As a non-native speaker I of course do not have an opinion on my own - those nuances are way too subtle for my sensitivity to even detect 😉

Hi, Rancher docs are a bit different as we actually use the MS style guide instead of the SUSE guide. This is a holdover from when Rancher was a separate company. The MS guide advises against "we" in most cases, while it encourages "you" in place of referring to the reader in the third person (as in "users should..." instead of "you should..."). It also strongly discourages passive voice. Contractions vs no contractions is probably the most obvious different from the SUSE style guide. The MS guide favors contractions, while the SUSE guide does't.

We did consider, as of late, switching to the SUSE guide, but decided against it for the time being. We already have an established docset with hundreds of pages, so standardizing those pages around another style guide would be a big task.

Reply:
That makes total sense, thanks for sharing this background! Will try to keep it in mind for the future.